py3status/modules/zypper_updates.py

```diff
@@ -0,0 +1,70 @@
+"""
+Display number of pending updates for OpenSUSE Linux.
+
+Display the system updates counted by zypper lu.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds
+        (default 600)
+    format: Display format to use
+        (default 'zypper: [\?color=update {update}]')
+
+Format placeholders:
+    {updates} number of pending zypper updates
+
+Color options:
+    color_bad: out to date system
+    color_degraded: Upgrade available
+    color_good: No upgrades needed
```

Comment:
Although it's technically not wrong because the `Color options` are specified inside `thresholds` config, this basically got replaced by `thresholds` option.

Instead of specifying three different colors, users would just had to specify a new `thresholds` config. 

```suggestion
Color thresholds:
    xxx: print a color based on the value of `xxx` placeholder
```

Reply:
done